osp/core/ontology/docs/dcat2.yml

```diff
@@ -10,23 +10,7 @@ namespaces:
     dcat2: http://www.w3.org/ns/dcat#
 
 active_relationships:
```

Comment:
@kysrpex, I added those active relations, since they were not shown in the `pretty_print`, if not assigned here in this file. I don't know if this is due to some specific querying in this or whether the `active_relationship` still plays a central role in osp-core.

```
Python 3.7.4 (default, Aug 13 2019, 20:35:49)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from osp.core.namespaces import dcat2
>>> a=dcat2.Distribution()
>>> b=dcat2.Resource()
>>> a.add(b, rel=dcat2.downloadURL)
<dcat.Resource: 8c21be84-e5d4-451e-934e-5d116e5bf821,  CoreSession: @0x7f84f2fffe50>
>>> from osp.core.utils import pretty_print
>>> pretty_print(a)
- Cuds object:
  uid: 3849ef62-18c5-4564-a640-659b99376670
  type: dcat.Distribution
  superclasses: dcat.Distribution
  description:
    http://www.w3.org/TR/vocab-dcat/
```

However, even without the `active`-specification, they seem to be findable with the `get`-methods of CUDS.

```
>>> a.get(rel=dcat2.downloadURL)
[<dcat.Resource: 8c21be84-e5d4-451e-934e-5d116e5bf821,  CoreSession: @0x7f84f2fffe50>]
>>> b.get(rel=dcat2.downloadURL.inverse)
[<dcat.Distribution: 3849ef62-18c5-4564-a640-659b99376670,  CoreSession: @0x7f84f2fffe50>]
```

Do you have an explaination for that?

Reply:
For other people that may be reading this: the question was answered privately. For people with access to [this document](https://owncloud.fraunhofer.de/index.php/f/390238000), the explanation is on slides 10-21.